collectors/charts.d.plugin/README.md

```diff
@@ -44,40 +43,39 @@ The variable `enable_all_charts` sets the default enable/disable state for all c
 
 ## A charts.d module
 
+> ### Note
```

Comment:
Move it to the introductory paragraph, not as a note, as a sentence. 
The following info should exist in the very beginning of all our docs. 
- This document does X
- To do Y, see LINK. 

Example (not ideal, but similar concept): 
<img width="735" alt="image" src="https://user-images.githubusercontent.com/43294513/231240127-efbc4356-5784-4be7-af30-1fdb3cac7bcc.png">

So every external plugin documentation should have this link to the Introduction, but at the very top.

Reply:
I addressed this in the latest commit